doc/lightning-withdraw.7.txt

```diff
@@ -0,0 +1,50 @@
+LIGHTNING-WITHDRAW(7)
+====================
+:doctype: manpage
+
+NAME
+----
+lightning-withdraw - Protocol for withdrawing funds from the 
+internal wallet.
+
+SYNOPSIS
+--------
+*withdraw* 'destination' 'satoshi'
+
+DESCRIPTION
+-----------
+The *withdraw* RPC command sends funds from c-lightning's 
+internal wallet to the address specified in 'destination'.
+
+The address can be of any Bitcoin accepted type, 
+including bech32.
+
+'satoshi' is the amount to be withdrawn from the internal 
+wallet (expressed, as name suggests, in satoshi)
```

Comment:
`'satoshi' can be the string '"all"' meaning to withdraw all available funds to the specified address, and to subtract fees from those funds.`